docs/install-config/download-installer.md

```diff
@@ -7,14 +7,14 @@ You download the Harbor installers from the [official releases](https://github.c
 
 - **Online installer:** The online installer downloads the Harbor images from Docker hub. For this reason, the installer is very small in size.
 
-- **Offline installer:** Use the offline installer if the host to which are are deploying Harbor does not have a connection to the Internet. The offline installer contains pre-built images, so it is larger than the online installer.
+- **Offline installer:** Use the offline installer if the host to which are deploying Harbor does not have a connection to the Internet. The offline installer contains pre-built images, so it is larger than the online installer.
 
-The installation processes are almost the same for both the online and offline installers.
+The installation processes are almost the same for online and offline installers.
```

Comment:
nit: I think you might still need the `the` in this sentence, `The installation processes are almost the same for the online and offline installers.`

Reply:
@a-mccarthy Good nit. It does read better with `the`. 

Resolved.